signalfx-org-metrics/README.md.jinja

```diff
@@ -67,21 +67,21 @@ These metrics track limits that Infrastructure Monitoring enforces for your orga
 * `sf.org.limit.hosts` (gauge): Maximum number of hosts that can send data to your organization.
    The limit is higher than your contractual limit to allow for burst and overage usage. If you
    exceed this limit, Infrastructure Monitoring drops data points from new hosts but keeps
-   data points for existing hosts. To monitor your usage against the limit, use the metric
+   accepting data points for existing hosts. To monitor your usage against the limit, use the metric
```

Comment:
hm... so does a user have to configure this metric in order to monitor their usage? or- just for one specific limit? If it is for all limits- I think this could come higher up the page and be called out as an action item the user has to take

Reply:
The limit is for the total number of hosts coming from all clients sending in data.